docs/CustomCapability.md

```diff
@@ -1 +1,104 @@
-## Coming Soon
+## Customizing the Capability Statement
+You now have the ability to generate a custom Capability Statement! Lets walk through how to do that. 
+
+### Creating a Statement Generator
+You can decide to include this code where you define your Asymmetrik Server configurations or include it in another file and simply require it in. For this purpose we will go with the latter option.
+
+First, we're going to require in the Asymmetrik FHIR Server source code. This will allow us to have access to the class that is responsible for generating the Capability Statement so that we can create our own.
+Then we'll create a function that will require in the CapabilityStatement class, and return a new instance that contains your information! Please make sure you remain compliant with the FHIR Specification.
+
+Next, we're going to create a custom Security Statement. This is apart of the Capability Statement. It provides information about the security of the current implementation. 
```

Comment:
Can we change the phrase "This is apart of the Capability Statement" to "This is separate from the Capability Statement." for clarity? The first time time I read it I mis-interpreted it as "this is a part of."  Thanks!